java/pom.xml

```diff
@@ -29,6 +29,41 @@
     <tag>HEAD</tag>
   </scm>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>6.4.0</version>
+        <executions>
+          <execution>
+            <id>bnd-process</id>
+            <goals>
+              <goal>bnd-process</goal>
+            </goals>
+            <configuration>
+            <bnd>
+            <![CDATA[ 
+            Export-Package: io.cucumber.gherkin.utils.*;-noimport:=true
+            Import-Package: *
```

Comment:
This looks incredibly fragile. While for a single package module it is fine. I imagine that if there are multiple packages, this gets messy quickly.

Also, would it be possible to extract this to a file instead?

Reply:
> This looks incredibly fragile. While for a single package module it is fine. 

I must confess I don't understand what part of it?

> Also, would it be possible to extract this to a file instead?

Yes one can extract it into a `bnd.bnd` file for example. I often find that a bit more disturbing for projects not using OSGi/bnd otherwise.